clang/lib/Tooling/DependencyScanning/DependencyScanningWorker.cpp

```diff
@@ -569,7 +569,8 @@ class DependencyScanningAction : public tooling::ToolAction {
 
       MDC = std::make_shared<ModuleDepCollector>(
           Service, std::move(Opts), ScanInstance, Consumer, Controller,
-          OriginalInvocation, std::move(PrebuiltModuleVFSMap));
+          OriginalInvocation, std::move(PrebuiltModuleVFSMap),
+          Service.getCompilerInvocationTransformation());
```

Comment:
The entire `Service` is being passed, no need to pass the transformation separately.